frontend/src/pages/tournaments/[id]/settings.tsx

```diff
@@ -161,7 +161,7 @@ function GeneralTournamentForm({
           values.dashboard_endpoint,
           values.players_can_be_in_multiple_teams,
           values.auto_assign_courts,
-          values.start_time.toISOString(),
+          new Date(values.start_time).toISOString(),
```

Comment:
this already works right?

If not, it should just be:

```suggestion
          values.start_time,
```